docs/modules/antibuild.mdx

```diff
@@ -6,4 +6,91 @@ title: Anti-Build
 This addon requires you to have a matching version of EssentialsX installed.
 :::
 
-## Frequently Asked Questions
\ No newline at end of file
+EssentialsX AntiBuild provides fine control over player interactions on your server through a permission-based system.
+You can specify which blocks and items players can use, restrict building entirely, and manage crafting and block
+interactions on a server-wide basis. It also features an alert system to notify admins when certain items are used.
+
+:::note
+EssentialsAntiBuild does not include any kind of region protection. To restrict building in specific regions we
+recommend installing [WorldGuard](https://enginehub.org/worldguard) instead.
+:::
+
+:::note
+We recommend you use [LuckPerms](https://luckperms.net/) with AntiBuild to make the most of its features.
+:::
+
+## Configuration
+You can access the EssentialsAntiBuild configuration by opening `/plugins/Essentials/config.yml`, and scrolling down
+until you see the section marked "EssentialsX AntiBuild".
+
+Control whether players without the `essentials.build` permission can build. Set to true to prevent building from
+players without this permission. Set to false to allow all players to build regardless of permissions.
+
+```yaml
+build: true
+```
+
+Control whether players without the `essentials.build` permission can use items. Set to true to prevent using items from
+players without this permission. Set to false to allow all players to use items regardless of permissions.
+
+```yaml
+use: true
+```
+
+EssentialsAntiBuild will warn players when they are not allowed to build. Set to false to disable.
+
+```yaml
+warn-on-build-disallow: true
+```
+
+Players with the `essentials.protect.alerts` permission will receive notifications when certain blocks trigger alerts.
+Specify the block types for which you want to receive alerts:
+
+`on-placement`: Blocks that will trigger alerts when placed.
+
+`on-use`: Items that will trigger alerts when used.
+
+`on-break`: Blocks that will trigger alerts when broken.
+
+:::info You can find a complete list of items [here](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Material.html).
+:::
+
+:::note
+Exempt users from triggering alerts with the `essentials.protect.alerts.notrigger` permission.
```

Comment:
I know this used to exist, but does it still?

Reply:
Yep. Still a thing.